scripts/microbe_stage/setup.lua

```diff
@@ -34,6 +34,7 @@ local function setupCamera()
     viewportEntity:addComponent(viewportComponent)
 end
 
+-- there must be some more robust way to script agents than having stuff all over the place.
```

Comment:
You could put them in their own scripts that are referenced in compounds.xml and dynamically loaded?